types/googlemaps/index.d.ts

```diff
@@ -3160,7 +3160,7 @@ declare namespace google.maps {
         }
 
         interface OpeningHours {
-            open_now: boolean;
```

Comment:
`open_now` still exists and is described in [the documentation][open_now]. Better mark it as [`@deprecated`](https://jsdoc.app/tags-deprecated.html).

```js
/** @deprecated */
open_now: boolean;
```

[open_now]: https://developers.google.com/maps/documentation/javascript/reference/places-service#PlaceOpeningHours.open_now